linker/shared/src/main/scala/org/scalajs/linker/backend/wasmemitter/LoaderContent.scala

```diff
@@ -189,10 +194,13 @@ export async function load(wasmFileURL, exportSetters, customJSHelpers) {
     "__scalaJSHelpers": myScalaJSHelpers,
     "__scalaJSExportSetters": exportSetters,
     "__scalaJSCustomHelpers": customJSHelpers,
```

Comment:
Nit: These module names should probably also go into `EmbeddedConstants` at this point. (I appreciate that we need `UTF8StringConstantsModule`, otherwise `""` everywhere would just be too confusing).

OK to do in a follow-up.

Reply:
I integrated the changes. I makes sense for it to go together.